portal-api/src/v1/facility-dates/requested-cover-start-date.js

```diff
@@ -1,30 +1,19 @@
-const moment = require('moment');
-const { hasValue } = require('../../utils/string');
+const { cloneDeep } = require('lodash');
+const { dateHasAllValues } = require('../validation/fields/date');
+const {
+  getDateAsEpochMillisecondString,
+  getStartOfDateFromDayMonthYearStringsReplicatingMoment
+} = require('../helpers/date');
 
-const getRequestedCoverStartDateValues = (facility) => {
+const hasAllRequestedCoverStartDateValues = (facility) => {
   const {
     'requestedCoverStartDate-day': requestedCoverStartDateDay,
     'requestedCoverStartDate-month': requestedCoverStartDateMonth,
     'requestedCoverStartDate-year': requestedCoverStartDateYear,
   } = facility;
 
-  return {
-    requestedCoverStartDateDay,
-    requestedCoverStartDateMonth,
-    requestedCoverStartDateYear,
-  };
-};
-
-const hasAllRequestedCoverStartDateValues = (facility) => {
-  const {
-    requestedCoverStartDateDay,
-    requestedCoverStartDateMonth,
-    requestedCoverStartDateYear,
-  } = getRequestedCoverStartDateValues(facility);
-
-  const hasRequestedCoverStartDate = (hasValue(requestedCoverStartDateDay)
-    && hasValue(requestedCoverStartDateMonth)
-    && hasValue(requestedCoverStartDateYear));
+  const hasRequestedCoverStartDate =
```

Comment:
we can just return this and no need for lines 18 to 22. If you want to keep it a named variable for readability, im happy, and we can just return that. (I know this is legacy code!)